### PR TITLE
create a contributing guide, update readmes

### PR DESCRIPTION
**Pull request by tyrasd**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Contributing
+
+Optimally, when contributing to this repository, please first discuss the change you wish to make via an [issue](https://github.com/GIScience/ohsome-examples/issues/new), [email](mailto:ohsome@heigit.org), or any other method with the owners of this repository before making a change.
```
Comment (joker234) [inline CONTRIBUTING.md:3]: Can the `owners of this repository` be linked to the GitHub team @GIScience/ohsome in any useful way? Or do you want to link to [ohsome.org](https://ohsome.org) or something similar?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+## Content and Aims of This Repository
+
+This _examples_ repository is meant to be a collection of curated examples for how to request and use the [ohsome-api](https://github.com/giscience/ohsome-api) effectively. The individual examples should be well documented, easy to follow and be composed of reusable building-blocks. Ideally, every example should be self-contained and not repeat concepts which are already shown in another example. The focus is more on showcasing the technical concepts and usage patterns rather than on the data analysis side of things (although it is of course best when it is possible to combine all of this).
+
+The project's readme also includes a list of examples where the ohsome API is used in real world applications and data analyses. Free free to propose to have your project added to the list.
```
Comment (joker234) [inline CONTRIBUTING.md:9]: Add a link to this list in the readme

**Review by joker234 — COMMENTED** [review]
Do you want to create/enhance the README within this PR by adding an initial list of other examples? `CONTRIBUTING.md` should be linked from the README as well.

**Comment by tyrasd** [comment]
> Do you want to create/enhance the README within this PR by adding an initial list of other examples? `CONTRIBUTING.md` should be linked from the README as well.

done

File: python/README.md [hunk 3]
```diff
@@ -1,9 +1,11 @@
...
 
 ## Jupyter Notebooks
+
+* [Data Aggregation Endpoint](https://nbviewer.jupyter.org/github/GIScience/ohsome-examples/blob/master/python/jupyter-notebooks/ohsome-data-aggregation.ipynb)
 * [Analysing HOT Tasking Manager Projects in Nepal](https://nbviewer.jupyter.org/github/GIScience/ohsome-examples/blob/master/python/jupyter-notebooks/ohsome_api_hot_tm_project1008.ipynb)
-* [ohsome API: Data Aggregation Endpoint](https://nbviewer.jupyter.org/github/GIScience/ohsome-examples/blob/master/python/jupyter-notebooks/ohsome-aggregation-plot.ipynb)
+* [Complex analysis with the magical filter parameter](https://nbviewer.jupyter.org/github/GIScience/ohsome-examples/blob/master/python/jupyter-notebooks/cycling-magic-filter-ohsomeAPI.ipynb)
 
 ## Documentation of Ressources:
 * [ohsome Rest-API](http://api.ohsome.org)
```
Comment (joker234) [inline python/README.md:11]: We never called the `ohsome API` `ohsome Rest-API`. Is there a reason it is called differently here?

File: README.md [hunk 4]
```diff
@@ -1,3 +1,25 @@
 # ohsome examples
 
-This Repository contains examples how to work with the ohsome api.
+This Repository contains a collection of examples of how to work with the [ohsome API](https://api.ohsome.org).
+
+## Python examples
+
+The [`python`](https://github.com/GIScience/ohsome-examples/tree/master/python#readme) directory contains the following examples written in Python using jupyter notebooks:
```
Comment (joker234) [inline README.md:7]: In all other places, `python` was large.

**Review by joker234 — APPROVED** [review]
Looks good to me, I just found two smaller issues that might be addressed here.

**Pull request merged**